### PR TITLE
Do not build and publish dist and activator zips

### PR DESCRIPTION
**Pull request by johanandren**
Refs  #22207

### CODE REVIEW

File: project/AkkaBuild.scala [hunk 1]
```diff
@@ -357,8 +348,7 @@ object AkkaBuild extends Build {
   val dontPublishSettings = Seq(
     publishSigned := (),
     publish := (),
-    publishArtifact in Compile := false,
```
Comment (patriknw) [inline project/AkkaBuild.scala]: sure that this is not needed?

**Review by patriknw — COMMENTED** [review]
you can also remove `scripts/publish_samples.sh`

File: project/AkkaBuild.scala (hunk 1 above)
Comment (johanandren) [inline project/AkkaBuild.scala]: Pretty sure it was added it was added to filter it out from the zip before, I'll double check.

**Comment by patriknw** [comment]
There is also some mention of s3 in the project/scripts/release

**Comment by johanandren** [comment]
Do'h, I removed those but lost them because didn't commit and ran the script to verify.

File: project/AkkaBuild.scala (hunk 1 above)
Comment (johanandren) [inline project/AkkaBuild.scala]: It's still in there, just github diff showing weirdly, see below.

**Review by patriknw — APPROVED** [review]

**Pull request merged**